xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/attachmentsjson.vm

```diff
@@ -105,7 +105,9 @@
     #if ($hasAccess)
       #set ($attachment = $doc.getAttachment($attachmentReference.name))
       #set ($authorReference = $services.model.resolveDocument($attachment.author))
+      ## The property "doc.fullName" is needed by LiveData since it is used to identify a live data entry.
```

Comment:
Well, if you use ``doc.fullName`` then all entries will have the same ID which is obviously not right. The ID should be the attachment name (if all attachments are from the same document). For this you should:
* add an ``id`` field below
* configure the ``entryDescriptor`` in the live data JSON configuration, see https://extensions.xwiki.org/xwiki/bin/view/Extension/Live%20Data%20Macro/#HLiveDataJSON

```
"entryDescriptor": {
  // The property that can be used to identify a live data entry. This is used for entry selection.
  "idProperty": "id",
},
```